src/annotator/components/AdderToolbar.js

```diff
@@ -3,6 +3,58 @@ import { LabeledButton, Icon } from '@hypothesis/frontend-shared';
 
 import { useShortcut } from '../../shared/shortcut';
 
+/**
+ * Render an inverted light-on-dark "pill" with the given `badgeCount`
+ * (annotation count). This is rendered instead of an icon on the toolbar
+ * button for "show"-ing associated annotations for the current selection.
+ *
+ * @param {object} props
+ *  @param {number} props.badgeCount
+ */
+function BadgeCountPill({ badgeCount }) {
```

Comment:
This name is a bit clunky. Given the intent is to fit the style of `Icon` components but display a numeric value instead of a graphic, maybe something like `<NumberIcon value={count}/>`?

Reply:
Happy to change. No attachment to current name.